Issue with nextjs since v0.3.7

I tried using @hcaptcha/react-hcaptcha with nextjs but the latest version that is working is 0.3.7.

Since 0.3.7 I am always getting the following error:
```
wait  - compiling / (client and server)...
error - ./node_modules/@hcaptcha/react-hcaptcha/dist/index.js:3:0
Module not found: Can't resolve '@babel/runtime/helpers/interopRequireDefault'

Import trace for requested module:
./pages/index.tsx

https://nextjs.org/docs/messages/module-not-found
wait  - compiling /_error (client and server)...
error - ./node_modules/@hcaptcha/react-hcaptcha/dist/index.js:3:0
Module not found: Can't resolve '@babel/runtime/helpers/interopRequireDefault'

Import trace for requested module:
./pages/index.tsx

https://nextjs.org/docs/messages/module-not-found
error - Error: Cannot find module '@babel/runtime/helpers/interopRequireDefault'
Require stack:
- /hcaptcha-repro/node_modules/@hcaptcha/react-hcaptcha/dist/index.js
- /hcaptcha-repro/.next/server/pages/index.js
- /hcaptcha-repro/node_modules/next/dist/server/require.js
- /hcaptcha-repro/node_modules/next/dist/server/load-components.js
- /hcaptcha-repro/node_modules/next/dist/build/utils.js
- /hcaptcha-repro/node_modules/next/dist/build/output/store.js
- /hcaptcha-repro/node_modules/next/dist/build/output/index.js
- /hcaptcha-repro/node_modules/next/dist/cli/next-dev.js
- /hcaptcha-repro/node_modules/next/dist/bin/next
```

Reproduction repository: https://github.com/Le0Developer/hcaptcha-repro

Commands to run: `npm run dev` (dev server) or `npm run build` (build static files)

--- a/.babelrc
+++ b/.babelrc
@@ -1,9 +1,9 @@
 {
   "presets": [
-    "@babel/env",
-    "@babel/react"
+    "@babel/preset-env",
+    "@babel/preset-react"
   ],
-  "plugins": [["@babel/transform-runtime", {
+  "plugins": [["@babel/plugin-transform-runtime", {
     "regenerator": true
   }]]
 }

--- a/package.json
+++ b/package.json
@@ -43,5 +43,8 @@
     "webpack": "^4.44.2",
     "webpack-cli": "^3.3.12",
     "webpack-dev-server": "^3.11.0"
+  },
+  "dependencies": {
+    "@babel/runtime": "^7.17.9"
   }
 }

